atlassian_jwt_auth/contrib/flask_app/utils.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+import jwt.exceptions
+import requests.exceptions
+
+
+def parse_jwt(verifier, encoded_jwt):
+    """Decode an encoded JWT using stored config."""
```

Comment:
I'd rather leave catching the errors to callers and note what they should be (exceptions).